src/components/base/JInputField/JInputField.js

```diff
@@ -0,0 +1,111 @@
+// @flow
+
+import React from 'react'
+import classNames from 'classnames'
+
+import { getErrorMessage } from 'utils/form'
+import { JFieldMessage } from 'components/base'
+
+import jInputFieldStyle from './jInputField.m.scss'
+
+export type JInputType = 'text' | 'password'
+export type JInputTheme = 'white'
+
+type Props = StyledComponent<JInputTheme> & {
+  disabled: boolean,
+  infoMessage: string,
+  input: FinalFormInput,
+  label: string,
+  meta: FinalFormMeta,
+  placeholder: string,
+  type: JInputType,
+  validateType: FinalFormValidateType,
+}
+
+function JInputField({
+  className,
+  theme,
+  disabled,
+  infoMessage,
```

Comment:
I guess, we don't need to set some message explicitly from outside

Reply:
It is input info message. We have this in design and we need it